<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choose_office_location</class>
 <widget class="QMainWindow" name="choose_office_location">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1954</width>
    <height>878</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(110, 204, 175);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1934</width>
         <height>815</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1920</width>
         <height>815</height>
        </size>
       </property>
       <widget class="QPushButton" name="search_for_patient_pushButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>370</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
        <property name="text">
         <string>Search For Patient</string>
        </property>
       </widget>
       <widget class="QLabel" name="logout_image_label">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>510</y>
          <width>181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Images/logout.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="choose_office_location_pushButton">
        <property name="geometry">
         <rect>
          <x>1190</x>
          <y>370</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
        <property name="text">
         <string>Choose Office Location</string>
        </property>
       </widget>
       <widget class="QLabel" name="choose_office_location_image_label">
        <property name="geometry">
         <rect>
          <x>970</x>
          <y>300</y>
          <width>181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Images/office.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_new_patient_pushButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>580</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
        <property name="text">
         <string>Add New Patient</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>170</y>
          <width>851</width>
          <height>651</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(173, 231, 146);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="frame_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>530</y>
           <width>231</width>
           <height>81</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(110, 204, 175);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>20</y>
           <width>91</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(110, 204, 175);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_5">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>560</y>
           <width>231</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(110, 204, 175);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>0</y>
           <width>71</width>
           <height>291</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(110, 204, 175);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_7">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>30</y>
           <width>51</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(110, 204, 175);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="search_for_patient_image_label">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>300</y>
          <width>181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Images/magnifying-glass.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logout_pushButton">
        <property name="geometry">
         <rect>
          <x>1190</x>
          <y>580</y>
          <width>211</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(249, 249, 249);</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>220</y>
          <width>1001</width>
          <height>541</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QLabel" name="add_new_patient_image_label">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>510</y>
          <width>181</width>
          <height>181</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(52, 77, 103);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Images/Images/plus-symbol.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <zorder>frame_6</zorder>
       <zorder>frame</zorder>
       <zorder>search_for_patient_pushButton</zorder>
       <zorder>logout_image_label</zorder>
       <zorder>choose_office_location_pushButton</zorder>
       <zorder>choose_office_location_image_label</zorder>
       <zorder>add_new_patient_pushButton</zorder>
       <zorder>search_for_patient_image_label</zorder>
       <zorder>logout_pushButton</zorder>
       <zorder>add_new_patient_image_label</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1954</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
